cms/djangoapps/contentstore/views/item.py

```diff
@@ -266,6 +269,8 @@ def service(self, block, service_name):
                 return DjangoXBlockUserService(self._user)
             if service_name == "studio_user_permissions":
                 return StudioPermissionsService(self._user)
+            if service_name == "lti-configuration":
+                return ConfigurationService(LTIConsumerHideFieldsFlag)
```

Comment:
Yep, ^ @cahrens 's assessment above matches my understanding.

BTW, I'm not a huge fan of putting code specific to one external XBlock into a generic app like `xblock_config` though I understand the need. So this particular use case looks like yet more motivation for OEP-6 (Context-scoped XBlock Fields) or moving to use django models (e.g. ConfigurationModel) directly from XBlocks.